server/tests/article.test.js

```diff
@@ -0,0 +1,167 @@
+import 'babel-polyfill'
+import path from 'path'
+import { ApolloServer, gql } from 'apollo-server'
+import { describe, expect, test } from '@jest/globals'
+import { importSchema } from 'graphql-import'
+import { createTestClient } from 'apollo-server-testing'
+import resolvers from './mock-resolvers'
```

Comment:
In order for this unit test to add value, it needs to import the real resolver from `../resolvers/articles/queries.js`. As discussed offline, let's pair program this on Monday to work through the issues with mocking axios.